components/combobox/combobox.jsx

```diff
@@ -254,6 +254,40 @@ const propTypes = {
 	 * Changes styles of the input. Currently `entity` is not supported. _Tested with snapshot testing._
 	 */
 	variant: PropTypes.oneOf(['base', 'inline-listbox', 'readonly']),
+	/**
+	 * Default value of input. Provide uncontroled behaviour
+	 */
+	defaultValue: PropTypes.string,
+	/**
+	 * Provided to input to make it disabled
+	 */
+	disabled: PropTypes.bool,
+	/**
+	 * Add loading spiner below the options
+	 */
+	loading: PropTypes.bool,
+	/**
+	 * Object for creating Add item below the options
+	 */
+	optionsAddItem: PropTypes.shape({
```

Comment:
@interactivellama Wouldn't it be better to include this as part of the options object and have a different `type` attribute ? Something like `option.type='header'` and `option.type='footer'` Same question goes for `optionsSearchEntity` ? Under the hood, it was much cleaner to "merge" those 2 props with `options` to get the same keyboard nav behavior and highlighting on mouse over. Plus, we wouldn't need to have 2 extra props.

Other question is: If we keep as is, I'd suggest having a more generic name: `optionsHeaderItem`  and `optionsFooterItems` ?

Also, should it accept an array instead of just one prop?

Reply:
I think the idea was that you wouldn't want to pass it through the filter like the other items on keypress--and that the data type is different since it's not data, it's UI chrome in a way.

As to type, I think I've seen multi-entity searching on Platform such as Accounts, Users, etc. so an array might make more sense. My original proposal called for arrays. I don't think I had caught it in the previous review. Good catch!

As to naming, I was hoping to keep it semantic just in case it ever changed places. such as Add Item moved to the top or something. Do you foresee different footer needs. As you say, it might be good to future proof it as a head/footer. I could go either way.